### PR TITLE
CLOUDP-116968: Increase client count and ignore find operation failures in serverless index stress workload

### PR DESCRIPTION
**Pull request by JamesHeppenstall**

### CODE REVIEW

File: src/workloads/serverless/IndexStress.yml [hunk 1]
```diff
@@ -28,6 +28,24 @@ Clients:
   Client4:
     QueryOptions:
       maxPoolSize: 250
+  Client5:
+    QueryOptions:
+      maxPoolSize: 250
```
Comment (betsybutton) [inline src/workloads/serverless/IndexStress.yml:33]: Was this the maxPoolSize when we ran it successfully yesterday? I remember decreasing it to 50 at some point, but I don't remember at what point..

**Review by betsybutton — COMMENTED** [review]
Thanks for getting this in! Just had one question :)

File: src/workloads/serverless/IndexStress.yml (hunk 1 above)
Comment (JamesHeppenstall) [reply src/workloads/serverless/IndexStress.yml:33]: I think we can safely keep the maxPoolSize at 250. As long as it's lower than 500 (which is the max connections per serverless instance), we won't hit any limits.

Comment (betsybutton) [reply src/workloads/serverless/IndexStress.yml:33]: Sounds good!

**Review by betsybutton — APPROVED** [review]
LGTM!!

**Comment by JamesHeppenstall** [comment]
evergreen merge

**Pull request merged**